src/WebJobs.Extensions.NotificationHubs/NotificationHubAttribute.cs

```diff
@@ -47,5 +47,11 @@ public sealed class NotificationHubAttribute : Attribute
         /// <see cref="NotificationHubsConfiguration"/>.
         /// </summary>
         public string HubName { get; set; }
+
+        /// <summary>
+        /// Optional. Boolean value to enable debug send on NotificationHubClient
+        /// <see cref="NotificationHubClient"/>.
+        /// </summary>
+        public bool EnableTestSend { get; set; }
```

Comment:
Exposing this setting at attribute level is being more explicit about turning on/off to debug sending push just that notification. We could add this as global setting, but we would have to add at attribute level as well as we have the ability to override global NH connection string and hub name.